### PR TITLE
Change TestNetwork to allow parallel execution

### PR DESCRIPTION
**Pull request by satnam6502**
Change the TestNetwork end to end test to support parallel executions. Remove dependency on JSON files. 
@filbranden

### CODE REVIEW

File: test/e2e/util.go [hunk 1]
```diff
@@ -54,6 +56,27 @@ func waitForPodRunning(c *client.Client, id string) {
...
+			glog.Warningf("Get pod %s failed: %v", podName, err)
+			continue
+		}
+		if pod.Status.Phase != api.PodPending {
+			glog.Infof("Saw pod %s out of pending state (found %q)", podName, pod.Status.Phase)
+			return true
+		}
+		glog.Infof("Waiting for pod %s status to be !%q (found %q)", podName, api.PodPending, pod.Status.Phase)
+	}
+	glog.Warningf("Gave up waiting for pod %s status to go out of pending", podName)
```
Comment (dchen1107) [inline test/e2e/util.go]: When we give up on waiting or eventual bailout, how about dumping pod.Status for debugging?

Comment (satnam6502) [inline test/e2e/util.go]: Good idea, but we should do this in a separate PR for improvements to the
util.go file.

S

**Comment by filbranden** [comment]
Looks good. I know it's Friday but I think it's worth merging it... I'll keep an eye on Jenkins, if it starts going really bad I'll revert.

Cheers,
Filipe

**Comment by satnam6502** [comment]
Thank you! And yes, let's revert immediately if this looks problematic.

**Comment by satnam6502** [comment]
First failure seems to be due to a rate limit issue. Let's see how the second test fares.

**Comment by filbranden** [comment]
Test 1477 succeeded and test 1476 failed in basic.sh and not in the Go tests... At the very least this one doesn't seem to make it worse, so let's see how the other runs over the weekend go.

**Comment by satnam6502** [comment]
Agreed.

**Comment by filbranden** [comment]
goe2e.sh also passed on 1478. Looks like there's progress.
